### ISSUE TITLE
Prototypes file needed for spade-ModernRP corpus

### ISSUE THREAD

**Issue by vannawillerton** [MontrealCorpusTools/PolyglotDB#127]
The prototypes file `spade-ModernRP_prototypes.csv` needs to be created and added to
https://github.com/MontrealCorpusTools/SPADE/tree/master/spade-ModernRP

@jeffmielke

**Comment by msonderegger** [comment]
@vannawillerton Jeff says he replied to these somewhere.. is that right / could you look?

**Comment by vannawillerton** [comment]
Yes he did, under spade-Edinburgh issue I believe. I used this info but 'true' prototype files still need to be made for each corpus so I left these issues up. Would you like me to take them down?

**Comment by msonderegger** [comment]
sorry for delay. I am not sure what he would like done and this is most relevant for him. Could you tag him and ask in spade-vowels, or on Basecamp?

**Comment by jeffmielke** [comment]
Hi. I just wanted to mention that I'm receiving these messages and I'm not
really sure what you guys are talking about.

Jeff
